tests/src/kakarot/test_state.py

```diff
@@ -71,8 +71,8 @@ def test_should_cache_precompiles(self, cairo_run):
             output = cairo_run("test__cache_precompiles")
             assert output == list(range(1, 10))
 
-        @SyscallHandler.patch("IERC20.balanceOf", lambda addr, data: [0, 1])
         @pytest.mark.parametrize("transaction", TRANSACTIONS)
```

Comment:
did it change something to reorder the decorators?

Reply:
no, reverting